<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labNum">
       <property name="text">
        <string>数量:</string>
       </property>
       <property name="buddy">
        <cstring>sBoxNum</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sBoxNum">
       <property name="suffix">
        <string>kg</string>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labPrice">
       <property name="text">
        <string>单价:</string>
       </property>
       <property name="buddy">
        <cstring>sBoxPrice</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnCalc">
       <property name="text">
        <string>计算总价</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labPriceSum">
       <property name="text">
        <string>总价:</string>
       </property>
       <property name="buddy">
        <cstring>sBoxPriceSum</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="sBoxPrice">
       <property name="prefix">
        <string>$</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>12.359999999999999</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="sBoxPriceSum">
       <property name="prefix">
        <string>$</string>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>19</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labDec">
       <property name="text">
        <string>十进制:</string>
       </property>
       <property name="buddy">
        <cstring>sBoxDec</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sBoxDec">
       <property name="prefix">
        <string>Dec </string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>13</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnDec">
       <property name="text">
        <string>转换为其他进制</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labBin">
       <property name="text">
        <string>二进制:</string>
       </property>
       <property name="buddy">
        <cstring>sBoxBin</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sBoxBin">
       <property name="prefix">
        <string>Bin </string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="displayIntegerBase">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnBin">
       <property name="text">
        <string>转换为其他进制</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labHex">
       <property name="text">
        <string>十六进制:</string>
       </property>
       <property name="buddy">
        <cstring>sBoxHex</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="sBoxHex">
       <property name="prefix">
        <string>Hex </string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnHex">
       <property name="text">
        <string>转换为其他进制</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
